tools/profiling/precice-profiling

```diff
@@ -518,6 +518,14 @@ def analyzeCommand(profilingfile, participant, event, outfile=None, unit="us"):
         .with_columns(pl.col("dur") * dur_factor)
     )
 
+    # Identify the global runtime (eid="_GLOBAL")
+    global_runtime = (
+        df.filter(pl.col("eid") == "_GLOBAL").select(pl.sum("dur")).to_numpy()[0][0]
+    )
```

Comment:
If we do `filter by eid` followed by `sort by rank`, then we should have all durations we need for computing this per-rank.

Below, we would have to `pl.col("dur") / global_runtime[pl.col("rank")]` or similar.

For parallel participants, we can sum this relative time just like we do with dur.